force-app/main/default/lwc/rd2Service/model.js

```diff
@@ -134,7 +135,7 @@ const getDonationValue = (state) => {
 
 const getChangeType = (state) => {
     if (isRecurringTypeChanged(state) || isNewRecord(state)) {
-        return "";
+        return state.changeType;
```

Comment:
Doing this allows a manual change to the picklist to "stick" even if the overall Value changed, which is in line with our current release.

Reply:
Yup, I thought about this last night after you reached out, I'll make the change and do a bit more testing!

Reply:
Just pushed a fix, thanks for adding that Jest test 👍